trinity/protocol/eth/requests.py

```diff
@@ -7,25 +8,29 @@
     BlockIdentifier,
     Hash32,
 )
+from p2p.protocol import BaseRequest
 
-from eth.rlp.headers import BlockHeader
-from eth.rlp.receipts import Receipt
-
-from p2p.exceptions import ValidationError
-
-from trinity.rlp.block_body import BlockBody
+from trinity.protocol.eth.constants import MAX_HEADERS_FETCH
 from trinity.protocol.common.requests import (
-    BaseRequest,
     BaseHeaderRequest,
 )
 
-from . import constants
+from .commands import (
+    BlockBodies,
+    BlockHeaders,
+    GetBlockBodies,
+    GetBlockHeaders,
+    GetNodeData,
+    GetReceipts,
+    NodeData,
+    Receipts,
+)
 
 
-class HeaderRequest(BaseHeaderRequest[Tuple[BlockHeader, ...]]):
+class HeaderRequest(BaseHeaderRequest):
     @property
     def max_size(self) -> int:
-        return constants.MAX_HEADERS_FETCH
+        return MAX_HEADERS_FETCH
```

Comment:
If you come back through here this could just be turned into a normal attribute of the class.